src/main/java/uiPages/DashboardPage.java

```diff
@@ -0,0 +1,30 @@
+package uiPages;
+
+import java.util.List;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import static helpers.WebElementHelper.areVisible;
+
+public class DashboardPage extends BasePage {
+
+  private final static int CAROUSELS_NR = 7;
+
+  @FindBy(className = "noo-container-fluid")
+  private List<WebElement> carousels;
+
+  @Override
+  protected boolean isCurrent() {
+    return areVisible(carousels.toArray(new WebElement[carousels.size()])) && carousels.size() == CAROUSELS_NR;
```

Comment:
This method needs more extra steps to be used in this form, when trying to verify a `List<WebElement>` size. It's a little bit of overhead every time. Maybe overload it to take a `List<WebElement>` as param? Or, create a new method, `verifyElementListSize(List<WebElement>, int size)`